packages/fx-core/src/common/tools.ts

```diff
@@ -495,3 +495,32 @@ export async function copyFiles(
     });
   }
 }
+
+export function getSiteNameFromResourceId(resourceId: string): string {
```

Comment:
should this be part of resource plugin's logic, since it's not so common? how do we decide which one should be put into common/tools.ts?

Reply:
This method can be used if resource plugin would like to parse "Microsoft.Web/sites" resource name from resource id. It is quite common. Not only function plugin can leverage it.